pages/en/contrib/Governance.md

```diff
@@ -20,18 +20,22 @@ Each repository has one or more lead maintainers responsible for:
 *   Seeking consensus on technical decisions.
 *   Making the final decisions when consensus cannot be achieved.
 
-Maintainers have npm publishing rights for modules and the final say on releasing new versions.
+Maintainers have npm publishing rights for modules and the final say on releasing new versions.  
 
-## Lead maintainers
+Refer to `CODEOWNERS` file in each repository for the list of lead maintainers. 
```

Comment:
Let's use the term "active maintainers" instead of "lead maintainers". We list community members in CODEOWNERS too.